components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/utils/LoggerUtils.java

```diff
@@ -169,4 +169,73 @@ private static Instant parseDateTime(String dateTimeString) {
         }
         return localDateTime;
     }
+
+    /**
+     * Util function to mask content.
+     *
+     * @param content Content that needs to be masked.
+     * @return masked content.
+     */
+    public static String maskContent(String content) {
+
+        if (StringUtils.isNotEmpty(content)) {
+            content = content.replaceAll(LogConstants.CONTENT_MASKING_REGEX, LogConstants.MASKING_CHARACTER);
+        }
+        return content;
+    }
+
+    /**
+     * Util function to mask claim values except userid claim.
+     *
+     * @param claims Map of user claims.
+     * @return sanitized map of user claims.
+     */
+    public static Map<String, String> maskClaimValues(Map<String, String> claims) {
+
+        Map<String, String> sanitizedClaims = new HashMap<>();
+        if (!claims.isEmpty()) {
+            for (Map.Entry<String, String> entry : claims.entrySet()) {
+                if (entry.getKey().equals(LogConstants.userIdClaimURI)) {
+                    sanitizedClaims.put(entry.getKey(), entry.getValue());
+                } else {
+                    sanitizedClaims.put(entry.getKey(), maskContent(entry.getValue()));
+                }
+            }
+        }
+        return sanitizedClaims;
+    }
+
+    /**
+     * Util function to mask claim value except userId claim when there is only one claim.
+     *
+     * @param claimURI   Claim URI.
+     * @param claimValue Claim Value that will be masked.
+     * @return masked claim value.
+     */
+    public static String maskClaimValue(String claimURI, String claimValue) {
+
+        String sanitizedClaimValue = claimValue;
+        if (StringUtils.isNotBlank(claimValue) && !claimURI.equals(LogConstants.userIdClaimURI)) {
+            sanitizedClaimValue = maskContent(claimValue);
+        }
+        return sanitizedClaimValue;
+    }
+
+    /**
+     * Util function to mask array of values.
+     *
+     * @param values Array of values need to be sanitized.
+     * @return sanitized array.
+     */
+    public static String[] sanitizeArraysOfValues(String[] values) {
+
+        if (LogConstants.isLogMaskingEnable) {
```

Comment:
in some utils methods, you have checked isLogMaskingEnable and in some it is not. better follow a consistent format